frontend/pages/index.js

```diff
@@ -627,7 +683,32 @@ const Homepage = () => {
                     {`Contact us on Discord to discuss your project and keep up with the latest updates on Moonstream Engine.`}
                   </chakra.span>
 
-                  <JoinDiscordButton />
+                  <RouteButton
```

Comment:
<img width="1748" alt="image" src="https://user-images.githubusercontent.com/61459744/169291018-d9efc5b7-9d0a-4a9a-9c2a-52dfca6a903e.png">

Can you make stuff centered and bigger so it looks better on larger screen? Since section is small it is actually OK to set grid item size to `50vh` on large screen so it does not bring so much empty space